src/plugins/plan/util.js

```diff
@@ -21,8 +21,8 @@
  *****************************************************************************/
 
 export function getValidatedData(domainObject) {
-    let sourceMap = domainObject.sourceMap;
-    let body = domainObject.selectFile?.body;
+    const sourceMap = domainObject.sourceMap;
```

Comment:
It could be, but I don't think it's needed? The availability of configurable Plan Views negates the need for multiple plan JSONs with differing lane visibility.

Reply:
Hmm maybe - I still like the idea of people having a baseline of what's hidden when the plan is first viewed.